src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -755,6 +755,37 @@ GUI_TEST_CLASS_DEFINITION(test_7384_2) {
     }
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7401) {
+    // 1. Open human_T1.fa.
+    GTFileDialog::openFile(os, dataDir + "samples/FASTA/human_T1.fa");
+
+    // 2. Select any part of sequence.
+    auto panView = GTUtilsSequenceView::getPanViewByNumber(os);
```

Comment:
nit: What is the type of PanView here: Is it a QWidget or PanView. I would suggest either using a type (it is short here) or name suffix: panViewWidget.

Reply:
OK